sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,237 @@
+
+### Fast Blocks
+
+The Stacks Nakamoto Upgrade, proposed in [SIP-021](https://github.com/stacksgov/sips/blob/main/sips/sip-021/sip-021-nakamoto.md#proposed-solution), enables fast blocks where user-submitted transactions will now take on the order of seconds, instead of tens of minutes. Thus, sBTC on Stacks Nakamoto will offer an improvement to Bitcoin’s current transaction times.
+
+The sBTC protocol not only addresses the limitations of Bitcoin's scripting system but also provides a secure and decentralized solution for utilizing Bitcoin in various applications.
+
+## Design
+
+While the first sBTC implementation is under development, the wrapped nature of the sBTC token means that any such system would have the following properties:
+
+- sBTC is a SIP-10 token backed 1:1 by BTC.
+- The sBTC peg wallet is maintained by the set of sBTC signers. These signers are responsible for the security and maintenance of the wallet, ensuring that sBTC is redeemable for BTC.
+- Bitcoin can be converted into sBTC within 3 Bitcoin blocks, and sBTC can be converted into Bitcoin within 6 Bitcoin blocks. sBTC relies on the forking behavior guaranteed by [SIP-021](https://github.com/stacksgov/sips/blob/main/sips/sip-021/sip-021-nakamoto.md) in order to maintain the peg wallet correctly across forks.
+
+## Specification
+
+Management of the sBTC peg wallet on the Bitcoin blockchain shall be managed by the proposed set of signers through a democratic process, involving the sBTC Signer Set rather than a single custodian. At launch, the sBTC protocol will be maintained by 15 independent entities that make up the sBTC Signer Set. The eligibility criteria to become an sBTC Signer are determined through the community governance process of ratifying this SIP.
+
+sBTC Signers are responsible for accepting or rejecting all sBTC deposit and withdrawal operations submitted to the network. For a transaction to be fulfilled, at least 70% of the signers need to approve the transaction. This means that the liveness and reliability of the signers is crucial to the success of the protocol. The system is live ("resilient") if at least 70% of the sBTC Signer voting power are online and honest. Then (and only then), deposits and withdrawals happen in a timely manner. The system is safe ("trustworthy") if at least 30% of the sBTC Signer voting power is honest. Then, no theft of funds can occur. Additionally, more details on sBTC deposit and withdrawals are included in the appendix of this SIP. 
+
+Throughout this release, sBTC will have an unchangeable and distinct signer set that will not explicitly be part of Stacks consensus. Given that there will be 15 unique signers that shall comprise the system, and each unique signer is allocated exactly one vote, it would require at least 11 out of 15 signatures for an sBTC operation to be fulfilled.
```

Comment:
I'm not sure this is true in the sBTC code.  It seems that signers can "vote in" other signers and "vote out" others if 11 of 15 signers vote to transfer the BTC to a new Bitcoin address, such that signature data from a _new, different_ signer set is required to spend it.

I don't have an opinion either way as to whether or not this is acceptable (the community will decide), but the SIP should be consistent with the implementation.

Reply:
Like, even if this isn't in the sBTC code as an explicit feature, the fact that sBTC supports private key rotation (mentioned below) is equivalent to sBTC supporting changing the signer set.  So unless this SIP mandates that the BTC deposit address is fixed forever, sBTC supports rotating signers.

Reply:
Ah, I see you have a subsection about upgrading the sBTC signer set below.  In that case, I think this paragraph needs to be revised.  The signer set is not "unchangeable".

Also, it is _not_ part of consensus.  No "explicitly" about it.

Reply:
Agreed. I've updated the SIP text to clarify that the sBTC signer set _is_  changeable and _not_ part of Stacks consensus.